src/widgets/MenuButton.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
```

Comment:
I'd move it into AppBar.tsx. It's not really a reusable widget, it's just a subcomponent of AppBar.

Reply:
I put it in its own file because it's also used in RootNavigator. We need it to tell the Appbar to display it, even if we can go back.

Reply:
Gotcha.